waspc/src/Wasp/JsImport.hs

```diff
@@ -0,0 +1,80 @@
+{-# LANGUAGE DeriveDataTypeable #-}
+
+module Wasp.JsImport
+  ( JsImport (..),
+    JsImportName (..),
+    JsImportData,
+    JsImportAlias,
+    JsImportPath,
+    getJsImportData,
+  )
+where
+
+import Data.Data (Data)
+import Data.List (isPrefixOf)
+import StrongPath (Dir', File', Path, Posix, Rel)
+import qualified StrongPath as SP
+
+data JsImport = JsImport
+  { -- | What is being imported.
+    _name :: JsImportName,
+    -- | Path from which we are importing.
+    _path :: JsImportPath,
+    -- | Alias for the imported name
+    _importAlias :: Maybe JsImportAlias
+  }
+  deriving (Show, Eq, Data)
+
+type JsImportPath = Path Posix (Rel Dir') File'
+
+type Identifier = String
+
+type JsImportAlias = String
+
+data JsImportName
+  = -- | Represents external imports like @import Identifier from "file.js"@
+    JsImportModule Identifier
+  | -- | Represents external imports like @import { Identifier } from "file.js"@
+    JsImportField Identifier
+  deriving (Show, Eq, Data)
+
+type JsImportIdentifier = String
```

Comment:
We have this, but we also have Identifier above. Maybe let's stick to just Identifier?

Reply:
I've kept only the `JsImportIdentifier`